src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+  };
+
+  calculator.add = function (value) {
+    return this.result + value;
+  }.bind(calculator);
```

Comment:
[CODE KNOWLEDGE] - Instead of using the `function` keyword and `.bind(calculator)`, you can use the method shorthand syntax. Also, you should use assignment with arithmetic operator to update the `result` property.

GOOD EXAMPLE:
add(value) {
  this.result += value;
},